examples/org.eclipse.swt.examples/META-INF/MANIFEST.MF

```diff
@@ -21,6 +21,9 @@ Export-Package: org.eclipse.swt.examples.accessibility,
  org.eclipse.swt.examples.layoutexample,
  org.eclipse.swt.examples.paint,
  org.eclipse.swt.examples.texteditor
-Require-Bundle: org.eclipse.swt
+Require-Bundle: org.eclipse.swt,
+ org.eclipse.ui,
+ org.eclipse.jface.text,
```

Comment:
better to not add cyclic dependencies. 
In this case I would add the example in jface repo.

Reply:
ok, dependency to jface and test app now removed with [ded8838]